pkg/tsdb/cloudwatch/metric_find_query.go

```diff
@@ -699,7 +699,7 @@ func (e *CloudWatchExecutor) ec2DescribeInstances(region string, filters []*ec2.
 			return !lastPage
 		})
 	if err != nil {
-		return nil, errors.New("Failed to call ec2:DescribeInstances")
+		return nil, fmt.Errorf("Failed to call ec2:DescribeInstances, %v", err)
```

Comment:
@marefr / @bergquist  Should this be `%w`, or maybe use errutil?

Reply:
Happy to change to using `%w`, although other functions in this file don’t currently do so. I can also update them if you like?

Reply:
Personally I prefer `errutil.Wrap` https://github.com/grafana/grafana/blob/master/pkg/util/errutil/errors.go#L13 but either works :)

@syscll updating all cases in this file would be great.

Reply:
I decided to stick with the stdlib as to remove the need for another package import. The `xerrors` pkg wrap functionality seems to be extremely similar to the new go1.13 wrapping feature - hope that's ok!